lametro/templates/lametro/event.html

```diff
@@ -219,20 +219,17 @@ <h4>Agenda</h4>
                         <div class="col-xs-4">
 
                         {% if event.start_time|compare_time %}
-                            {% if 'board meeting' in event.name|lower or 'la safe' in event.name|lower %}
-                            <h4>Minutes</h4>
+                        <h4>Minutes</h4>
```

Comment:
Can you indent this one level, so it's offset from the opening `if event.start_time...`?